package.json

```
@@ -42,6 +42,7 @@
       "eslint --fix",
       "prettier --write"
     ],
+    "*.md": "eslint --fix -c eslint.config-content.js",
```

```suggestion
    "*.md": [
      "eslint --fix",
      "eslint --fix -c eslint.config-content.js"
    ],
```

Since we're also linting code blocks in .md files, maybe it should be included here?

---

Thanks for the suggestion! I agree it's a better solution.

I've added a new commit in https://github.com/eslint/markdown/pull/343/commits/c57a02b54359687490fa8de7aadcfe31ba14cb51.